src/ledger/parse/check-create.ts

```diff
@@ -0,0 +1,29 @@
+import * as assert from 'assert'
+import {parseTimestamp} from './utils'
+import {removeUndefined} from '../../common'
+import parseAmount from './amount'
+
+function parseCheckCreate(tx: any): Object {
```

Comment:
We're moving towards typing the output of these parsing functions so that they're easier to consume (also a great place to document the formatted type). 

So instead of returning a type `Object` you would return a type `FormattedCheckCreate`, something like this:

```ts
export type FormattedCheckCreate = {
  // account that can cash the check.
  destination: string,
  // amount the check is allowed to debit the sender, including transfer fees on non-XRP currencies.
  sendMax: Amount,
  // (Optional) identifies the reason for the check, or a hosted recipient.
  destinationTag?: string,
  // (Optional) time in seconds since the Ripple Epoch.
  expiration?: string, 
  // (Optional) 256-bit hash representing a specific reason or identifier.
  invoiceID?: string
}

function parseCheckCreate(tx: any): FormattedCheckCreate {
  // ...
}
```

Can you do this for the other two new parsers?